actions/class.RestQtiTests.php

```diff
@@ -23,6 +23,8 @@
 use oat\taoQtiTest\helpers\QtiPackageExporter;
 use oat\taoQtiTest\models\tasks\ImportQtiTest;
 use oat\taoQtiItem\controller\AbstractRestQti;
+use core_kernel_classes_Resource as Resource;
+use oat\taoTests\models\MissingTestmodelException;
```

Comment:
I see it has a bad readme file, could we add an example from PR description to the readme?

Reply:
didn't get it. could you provide a bit more context to your comment?

Reply:
if you mean request example i think in readme it covered by link to [OpenApi doc](https://github.com/oat-sa/extension-tao-testqti#rest-api)

Reply:
Sorry, it nothing to related for that line I talked about README.md file

Reply:
I just follow by link and README.md and somehow it wasn't. work for me

Reply:
just updated readme url by rawgithubcontent url